src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -74,60 +76,138 @@
+            <th>{{ 'Value'|trans({}, 'Admin.Global') }}</th>
+            <th class="text-right">{{ 'Actions'|trans({}, 'Admin.Global') }}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for discount in orderForViewing.discounts.discounts %}
+            <tr>
+              <td>{{ discount.name }}</td>
+              <td>
+                {% if discount.amountRaw %}-{% endif %}
+                {{ discount.amountFormatted }}
+              </td>
+              <td class="text-right">
+                <form action="{{ path('admin_orders_remove_cart_rule', {'orderId': orderForViewing.id, 'orderCartRuleId': discount.cartRuleId}) }}" method="post">
+                  <button type="submit"
+                          class="ml-2 tooltip-link btn btn-text"
+                          data-toggle="pstooltip"
+                          data-placement="top"
+                          data-original-title="{{ 'Delete'|trans({}, 'Admin.Actions') }}"
+                  >
+                    <i class="material-icons">delete</i>
+                  </button>
+                </form>
+              </td>
+            </tr>
+          {% endfor %}
+        </tbody>
+      </table>
+    {% endif %}
+
     <div class="row">
       <div class="col-md-12">
-        <div style="background-color: #F8F8F8; padding-top: 18px; padding-left: 18px; padding-right: 18px; padding-bottom: 16px;">
+        <div class="info-block">
           <div class="row">
             <div class="col-sm text-center">
               <p class="text-muted mb-0"><strong>{{ 'Products'|trans({}, 'Admin.Global') }}</strong></p>
-              <strong>14.99</strong>
-            </div>
-            <div class="col-sm text-center">
-              <p class="text-muted mb-0"><strong>{{ 'Shipping'|trans({}, 'Admin.Catalog.Feature') }}</strong></p>
-              <strong>99.99</strong>
+              <strong>{{ orderForViewing.prices.productsPriceFormatted }}</strong>
             </div>
+
+            {% if orderForViewing.prices.discountsAmountRaw %}
+              <div class="col-sm text-center">
+                <p class="text-muted mb-0"><strong>{{ 'Discounts'|trans({}, 'Admin.Orderscustomers.Feature') }}</strong></p>
+                <strong>-{{ orderForViewing.prices.discountsAmountFormatted }}</strong>
+              </div>
+            {% endif %}
+
+            {% if orderForViewing.prices.wrappingPriceRaw %}
+              <div class="col-sm text-center">
+                <p class="text-muted mb-0"><strong>{{ 'Wrapping'|trans({}, 'Admin.Orderscustomers.Feature') }}</strong></p>
+                <strong>{{ orderForViewing.prices.wrappingPriceFormatted }}</strong>
+              </div>
+            {% endif %}
+
+            {% if orderForViewing.prices.shippingPriceRaw %}
+              <div class="col-sm text-center">
+                <p class="text-muted mb-0"><strong>{{ 'Shipping'|trans({}, 'Admin.Catalog.Feature') }}</strong></p>
```

Comment:
cf. https://github.com/PrestaShop/PrestaShop/pull/15388#discussion_r329576339

Reply:
Are we talking of the same thing here?

Reply:
Nope, it's different, it's order summary line called "Shipping" that displays shipping price.